swagger_server/utils/db_utils.py

```diff
@@ -1,64 +1,51 @@
 import logging
 import os
 
-import _pickle as pickle
-import dataset
+import pymongo
 
+DB_NAME = os.environ.get("DB_NAME")
 DB_CONFIG_TABLE_NAME = os.environ.get("DB_CONFIG_TABLE_NAME")
+MONGODB_CONNSTRING = os.environ.get("MONGODB_CONNSTRING")
```

Comment:
Also, similar to https://github.com/atlanticwave-sdx/sdx-controller/pull/126 review comment, maybe it is useful to form the connection string with `f"mongodb://{MONGO_USER}:{MONGO_PASSWORD}@{MONGO_HOST}:{MONGO_PORT}/"`?

It probably is better/nicer to be consistent with MQ configuration?  Maybe it is a matter of opinion.

Reply:
Yeah, both ways should work the same, I think we can stick with the MONGODB_CONNSTRING for now. Because some fields, such as MONGO_PORT may not exist , so the format will break.

Reply:
I suggested using something like `MONGO_PORT = os.environ.get("MONGO_PORT", 27017)` in the other review, although this should be fine for now.